README.md

```diff
@@ -16,8 +16,16 @@ the square brackets (for multiple-choice questions).
 What are constructor functions and the `new` keyword? What is a prototype? Describe an example of when we would use a constructor function versus a prototype.
 
 Your Answer:
-```text
-
+```
+constructor functions are functions that are used to build an object.  
+They allow programmers to make multiple instances of an object.  
+Using the 'new' keyword when calling a constructor functions creates an instance
+of an object.
+Every object in Javascript has a prototype property that points to another object.
+Methods and properties defined on the prototpe property are available to all the objects made by the constructor.
+
+We use a constructor to make objects that share the same properties and methods.
+We use a prototype to define properties and mehtods that not all object instances will use.
```

Comment:
Yes, except the last part could be considered the opposite. We would want define properties and methods on the prototype that all instances would use and share. This way we can eliminate duplicate code from being created every time a new instance is created using a constructor function.